volume17.xml

```diff
@@ -0,0 +1,215 @@
+                </dateline>
+                <p><persName>El&#232;na</persName> arrived from <placeName ref="Khartoum">Khartoum</placeName> last evening. Mr. and <persName>Mrs. Newberry</persName> and <persName>Mr. Johnson</persName> (P&#232;re) <persName>Mr. Whymper</persName> were here for tea. <persName>Mr. Nicol</persName> to dine. We started 10 minutes to 11 o'clock &#8211; and at 6 p.m. were at <placeName>Keneh,</placeName> where we waited for <persName>Harry</persName> and <persName>El&#232;na</persName> to visit <placeName>Dendera</placeName> &#8211; then we moved on 2-3 miles for the night &#8211; making altogether 50 miles.</p>
+                <dateline>
+                    <date when="1911-03-04">Saturday. Mar. 4th.</date>
+                </dateline>
+                <p>40 miles. Sailed at 7 A.M. &#8211; reached <persName>Nagh Hamadi</persName> at 12:15, passed the bridge and then waited until 3:30, waiting for the tug to cool &#8211; and reached <persName>Balliana</persName> at 5:30 where we wait until tomorrow &#8211; to give <persName>El&#232;na Rathbone</persName> a chance to see <persName>Abydos</persName> &#8211; and we promised the <name>Navilles</name> to lunch with them. It has been a fine, warm day &#8211; and our brillian young moon, and the resplendent <persName>Venus,</persName> make the nights too beautiful to sleep.</p>
+                <dateline>
+                    <date when="1911-03-06">Monday. Mar. 6th.</date>
+                </dateline>
+                <p>Started about 10 for <persName>Abydos</persName> &#8211; fearing a hot day &#8211; had 2 carriages &#8211; we found a cold north wind blowing, and we really suffered with cold. It was a market day, and the people were streaming along the dusty road &#8211; men, women carrying babies, camels, donkeys, sheep and <pb n="81"/> goats &#8211; making the air thick with dust. It is such a pity that one must always see <persName>Abydos</persName> in the glare of mid-day! Mme. <persName>Naville</persName> came up from their little house in its nest in the sandy desert, to see us &#8211; and we all went back with her &#8211; I sitting on a man's saddle on a wretched donkey, - the cold wind almost blowing my clothes off &#8211; and my eyes blinded. But the little house was cheerful and comfortable and the lunch very good, and <persName>M.</persName> and Mme. <persName>Naville</persName> very nice as they always are. Mr. <persName>Whitmore</persName> and <persName>Mr. Dixon,</persName> young men working with the <orgName>Navilles.</orgName> The ride home was away from the wind, and the fields so green and fragrant with their blossoming crops, and the birds singing everywhere. The people were in their straw houses, which they build for watching their crops &#8211; and it was very pastoral. Found a huge mail waiting for us. We steamed down to <persName>Girga,</persName> 8 miles, and are here for the night.</p>
```

Comment:
Abydos tag changed from pers to place. Girga tag changed from pers to place with reference to "Girgeh." Multiple pers refs needed